fibbingnode/res/templates/ospf.mako

```diff
@@ -35,4 +35,9 @@ router ospf
   % for itf in node.ospf.passive_interfaces:
   passive-interface ${itf.name}
   % endfor
+  % if node.ospf.throttling and node.ospf.lsa:
+  timers throttle spf ${node.ospf.throttling['spf'].get('delay')} ${node.ospf.throttling['spf'].get('initial_holdtime')} ${node.ospf.throttling['spf'].get('max_holdtime')}
+  timers throttle lsa all ${node.ospf.throttling['lsa-all'].get('min_ls_interval')}
```

Comment:
If you use the ConfigDict() as requested above, you can then simply this to
node.ospf.throttling.spf.delay